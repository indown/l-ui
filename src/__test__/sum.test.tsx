// import sum from '../sum'

// test('adds 1 + 2 to equal 3', () => {
//   expect(sum(1, 2)).toBe(3);
// });

import Sum from '../sum'
import React from 'react';
import renderer from 'react-test-renderer';

test('Sum is a react component!',() => {
  const component = renderer.create(
    <Sum></Sum>
  );
  expect(component.toJSON()).toMatchSnapshot()
})