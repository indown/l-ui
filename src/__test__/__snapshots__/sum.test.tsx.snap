// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Sum is a react component! 1`] = `
<div>
  11111
</div>
`;

exports[`div 1`] = `
<div>
  11111
</div>
`;
